### ISSUE TITLE
editing config.yml has no effect

### ISSUE THREAD

**Issue by Armed-Hippie** [Telesphoreo/TotalFreedomMod#2]
if you edit config.yml in the totalfreedommod folder when it is installed on your server, the changes will not apply.

**Comment by Telesphoreo** [comment]
You need to use /tfm reload or restart the server.

**Comment by Armed-Hippie** [comment]
nope, it only works when i recompile it with a different config.yml file on the inside
